libs/remix-ui/debugger-ui/src/lib/vm-debugger/global-variables.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react' // eslint-disable-line
 import DropdownPanel from './dropdown-panel' // eslint-disable-line
 import { BN } from 'bn.js'
-import Web3 from 'web3'
+import { toBN } from 'web3-utils-legacy'
```

Comment:
This was a temporary decision made to reduce the friction of migrating to `4.x` since `BN` is no longer used

Reply:
I've removed this dependency